sherpa/astro/ui/utils.py

```diff
@@ -2410,10 +2418,10 @@ def set_syserror(self, id, val=None, fractional=False, bkg_id=None):
         val : array or scalar
            The systematic error.
         fractional : bool, optional
-           If ``False`` (the default value), then the ``val`` parameter is
```

Comment:
The single and double tick marks have a different meaning: `foo` is a link/reference whereas ``foo`` is a "code fragment"; see http://www.sphinx-doc.org/en/stable/rest.html and links therein. We could actually have None/False/... as links, as they could link to the actual Python documentation for those items, but it's not clear whether it's worth it. So we want ``get_dep() * val`` to remain double quotes. 

The trouble with `foo` is that the documentation system can get confused and dump a warning about  an invalid link. I haven't really seen that with my latest attempt at the docs (#302), but I think I saw it in previous attempts.

There's also "tweaks" to the rules. For instance, the astropy (essentially the NumPy ones) docstring rules, which I'm trying to follow, say to use `foo` when referring to parameter values of a function [*], but it's not 100% clear if that is only in the parameter block or in the whole method docstring.

[*] see section 4 of http://docs.astropy.org/en/stable/development/docrules.html#sections

I admit that the current documentation is a mix. I am trying to move towards a consistent style, but haven't ironed out the edges yet.